src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
-    { I18N_CLIENT_LOGGED_OUT, "You logged out" },
-    { I18N_CLIENT_PING_TIMEOUT, "You logged out (ping timeout)" },
-    { I18N_CLIENT_TOO_MANY_RESENDS, "You logged out (too many resends)" },
-    { I18N_CLIENT_APPLICATION_LAYER, "You logged out (application layer quit)" },
-    { I18N_CLIENT_SERVER_MESSAGE, "Server Message" },
-    { I18N_CLIENT_MAP_INFO, "map: ${1}" },
-    { I18N_CLIENT_JOIN_REFUSED, "YOUR JOIN REQUEST WAS REFUSED" },
-    { I18N_CLIENT_FRIENDLY_FIRE, "FRIENDLY FIRE: WATCH OUT!!!" },
-    { I18N_CLIENT_UNBALANCED, "GAMEPLAY IS UNBALANCED" },
-    { I18N_CLIENT_WARMUP, "please warm up" },
-    { I18N_CLIENT_GAME_BEGINS, "game begins" },
-    { I18N_CLIENT_GAME_OVER, "GAME IS OVER" },
-    { I18N_CLIENT_RENAME, "${1} is now known as ${2}" },
-    { I18N_CLIENT_SKIN_CHANGE, "${1} changed the skin to ${2}" },
+    { I18N_CLIENT_TRANSFER, "Transferre ${1} (${2}%)" },
+    { I18N_CLIENT_ENTER_MSG, "Écrire un message" },
+    { I18N_CLIENT_CONNECTING_TO1, "Se connecte à ${1}, ping = ${2}" },
+    { I18N_CLIENT_CONNECTING_TO2, ", demande un mot de passe" },
+    { I18N_CLIENT_CONNECTING_TO3, "(joueurs: ${1}/${2})" },
+    { I18N_CLIENT_LOGGED_IN, "vous êtes connecté" },
+    { I18N_CLIENT_LOGGED_OUT, "vous êtes déconnecté" },
+    { I18N_CLIENT_PING_TIMEOUT, "Vous êtes déconnecté (la session a expirée)" },
+    { I18N_CLIENT_TOO_MANY_RESENDS, "Vous êtes déconnecté (trop de renvois)" },
+    { I18N_CLIENT_APPLICATION_LAYER, "Vous êtes déconnecté (application layer quit)" },
+    { I18N_CLIENT_SERVER_MESSAGE, "Message du serveur" },
+    { I18N_CLIENT_MAP_INFO, "carte: ${1}" },
+    { I18N_CLIENT_JOIN_REFUSED, "VOTRE REQUÊTE DE CONNECTION A ÉTÉ REFUSÉE" },
+    { I18N_CLIENT_FRIENDLY_FIRE, "TIR AMI: ATTENTION!!!" },
+    { I18N_CLIENT_UNBALANCED, "JEU NON ÉQUILIBRÉ" },
+    { I18N_CLIENT_WARMUP, "merci de vous échauffer" },
+    { I18N_CLIENT_GAME_BEGINS, "la partie commence" },
+    { I18N_CLIENT_GAME_OVER, "LA PARTIE EST FINIE" },
+    { I18N_CLIENT_RENAME, "${1} est maintenant connu comme ${2}" },
+    { I18N_CLIENT_SKIN_CHANGE, "${1} à changer son personnage pour ${2}" },
 
-    { I18N_TNMT_NEW_MAP_LOADED, "new map loaded" },
-    { I18N_TNMT_SPECTATING, "SPECTATING" },
-    { I18N_TNMT_WARMUP, "WARM UP" },
-    { I18N_TNMT_SB_CTC_TITLE, "CATCH THE COIN" },
-    { I18N_TNMT_SB_PLAYER, "PLAYER" },
+    { I18N_TNMT_NEW_MAP_LOADED, "nouvelle carte chargée" },
+    { I18N_TNMT_SPECTATING, "SPECTATEUR" },
+    { I18N_TNMT_WARMUP, "ÉCHAUFFEMENT" },
+    { I18N_TNMT_SB_CTC_TITLE, "ATTRAPEZ LA PIÈCE" },
+    { I18N_TNMT_SB_PLAYER, "JOUEUR" },
     { I18N_TNMT_SB_TOTAL, "TOTAL" },
     { I18N_TNMT_SB_PING, "PING" },
-    { I18N_TNMT_SB_SPECTATORS, "SPECTACTORS:" },
-    { I18N_TNMT_SB_DM_TITLE, "DEATHMATCH" },
+    { I18N_TNMT_SB_SPECTATORS, "SPECTATEURS:" },
+    { I18N_TNMT_SB_DM_TITLE, "MATCH À MORT" },
     { I18N_TNMT_SB_FRAGS, "FRAGS" },
-    { I18N_TNMT_SB_KILLS, "KILLS" },
+    { I18N_TNMT_SB_KILLS, "TUÉS" },
```

Comment:
MORTS would be more understood as DEATHS, which would mean the number of time you died. Is that the right meaning for it ? If so, then the game is inconsistent in its basis locale :p